vyper/venom/passes/sccp/sccp.py

```diff
@@ -329,6 +333,28 @@ def _replace_constants(self, inst: IRInstruction):
                 if isinstance(lat, IRLiteral):
                     inst.operands[i] = lat
 
+    def _fix_phi_nodes(self):
```

Comment:
Don't we know where the possible degenerate `phi`s might be at the time of replacing `jnz` to `jmp`?

Reply:
I did some modification on that in new commits